lexicons/com/atproto/sync/getRecord.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lexicon": 1,
+  "id": "com.atproto.sync.getRecord",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Gets blocks needed for existence or non-existence of record.",
+      "parameters": {
+        "type": "params",
+        "required": ["did", "collection", "rkey"],
+        "properties": {
+          "did": {"type": "string", "description": "The DID of the repo."},
+          "collection": {"type": "string" },
+          "rkey": {"type": "string" },
+          "commit": {"type": "string", "description": "An optional past commit CID."}
+        }
+      },
+      "output": {
+        "encoding": "application/cbor"
```

Comment:
Any thoughts on whether we should consider `application/vnd.ipld.car` for this?  Popped into my head, looking at these re: mimes of blocks and CARs: https://github.com/ipfs/in-web-browsers/issues/192 https://github.com/ipfs/kubo/pull/8758 https://www.iana.org/assignments/media-types/application/vnd.ipld.car .

Reply:
yup great point

fixed here: https://github.com/bluesky-social/atproto/pull/485